cluster/sync.sh

```diff
@@ -28,3 +28,5 @@ for i in $(seq 1 ${KUBEVIRT_NUM_NODES}); do
 done
 
 ./cluster/kubectl.sh apply -f ./manifests/generated/cdi-controller.yaml
+# this http server is used by the functional tests
+./cluster/kubectl.sh apply -f ./manifests/generated/disks-images-provider.yaml
```

Comment:
how does this fit in with the server provided by pr #315 ?

Reply:
This is a separate approach. I use a container with nginx in it to serve the static iso files rather that writing code to do that. 

It's hard for me to justify using maintaining code for an http server when we can accomplish the same thing with little effort using nginx. Is there any benefit to the http server in #315 over using nginx? Maybe I'm missing something?

Reply:
@davidvossel I think the difference here, is that #315 uses one source file, and it generates all combinations of file formats we need, eg: tar/tar.gz/raw/etc for testing. We could do the same thing with nginx if we have an instance for each combination that it can serve. The difference would be some work to create all the files, ahead of time vs work to maintain the file server to support all the different format.

Reply:
@davidvossel Fair point - I don't know why I hadn't considered using nginx.  

As for the generation of files, that code exists as a separate binary to be run inside an init container.  It uses the same FormatTestData process from our unit tests, which keeps the addition of new code minimal.  The way I've implemented it, there are 3 containers - init, http, and s3.  They share access to the generated files through a shared emptyDir volume, so there's no duplication required.

Do you think the init process would be useful in the context of this PR?  If so, i can scrap the http server in favor of your implementation here.

Reply:
@copejon ah, yes i understand better what's going on now.

Yeah, I'd suggest pulling in the nginx server approach i took here into your work. Once your PR is merged, I'll scrap the http server in this PR and leverage what you've provided.  Sound good?